avocado/core/test.py

```diff
@@ -741,7 +741,7 @@ def _run_avocado(self):
             raise exceptions.TestSkipError(details)
         except exceptions.TestCancel as details:
             stacktrace.log_exc_info(sys.exc_info(), logger=LOG_JOB)
-            raise
+            raise exceptions.TestCancel(details)
```

Comment:
Here, the only reason we are raising the same exception again, it is just because of the log. With an explicit raise exception, this will raise the same exception twice ("During handling of the above exception, another exception occurred"). Not wrong, but increases the stack trace and it is slightly different from the current behavior.

My suggestion here is to use remove the `details` var:

```python
except exceptions.TestCancel:
      stacktrace.log_exc_info(sys.exc_info(), logger=LOG_JOB)
      raise
```

Reply:
@beraldoleal ack, that would mean reverting this change and instead 'fixing' this with a pylint override.

Reply:
> @beraldoleal ack, that would mean reverting this change and instead 'fixing' this with a pylint override.

Just force-pushed an update with an override.